microsoft-edge/progressive-web-apps-chromium/how-to/widgets.md

```diff
@@ -28,19 +28,29 @@ In Windows 11, Progressive Web Apps (PWAs) can define widgets, update them, and
 
 Widgets are experimentally supported starting with Microsoft Edge 108.
 
-To enable widget support in Microsoft Edge for your website:
+#### Enable local development of widgets
 
-1. Enroll your site in the [Web App Widgets](https://developer.microsoft.com/microsoft-edge/origin-trials/web-app-widgets/registration/) origin trial. To learn more about origin trials, see [how to enroll your site in an origin trial](origin-trials.md#enroll-your-site-in-an-origin-trial).
+To enable local development of widgets:
+
+1. Install [WinAppSDK 1.2](/windows/apps/windows-app-sdk/older-downloads#windows-app-sdk-12).
 
 1. In Microsoft Edge, go to `edge://flags`.
 
-1. In the **Search flags** text field, type **Web App widgets**.
+1. In the **Search flags** text field, type **Web App widgets**, and in the **Web App widgets** dropdown menu, select **Enabled**.
+
+   ![Enabling the Web App widgets flag in the edge://flags page](../media/widgets-flag.png)
 
-1. In the **Web App widgets** dropdown menu, select **Enabled**.
+1. Next, use the **Search flags** text field again, type **Experimental Web Platform Features**, and in the **Experimental Web Platform Features** dropdown menu, select **Enabled**.
 
-1. Click the **Restart** button which appears in the lower right:
+   ![Enabling the Experimental Web Platform Features flag in the edge://flags page](../media/experimental-features-flag.png)
 
-   ![Enabling the Web App widgets flag in the edge://flags page](../media/widgets-flag.png)
+1. Click the **Restart** button which appears in the lower right.
+
+#### Enable widgets for your users
+
+To enable widget support in Microsoft Edge for users of your production PWA, enroll your site in the [Web App Widgets](https://developer.microsoft.com/microsoft-edge/origin-trials/web-app-widgets/registration/) origin trial.
+
+To learn more about origin trials, see [how to enroll your site in an origin trial](origin-trials.md#enroll-your-site-in-an-origin-trial).
```

Comment:
```suggestion
To learn more about origin trials, see [how to enroll your site in an origin trial](./origin-trials.md#enroll-your-site-in-an-origin-trial).
```
https://learn.microsoft.com/microsoft-edge/progressive-web-apps-chromium/how-to/origin-trials

optional - not to provide complicated tagging, but b/c enables Autocompletion and this is the tagging inserted by Learn Auth Pack (Alt+M) in VS Code.